lib/projects.js

```diff
@@ -41,6 +41,10 @@ var projects = {
   "webpack-cli": {
     name: 'webpack-cli',
```

Comment:
Since the module on npm uses "webpack CLI" let's use that 😄 

```suggestion
    name: 'webpack CLI',
```